package.json

```diff
@@ -6,46 +6,33 @@
   "license": "GPL-3.0-or-later",
   "repository": "ElrondNetwork/dapp-core",
   "main": "dist/index.js",
-  "module": "dist/dapp-core.esm.js",
-  "typings": "dist/index.d.ts",
+  "typings": "dist/hooks/index.d.ts",
```

Comment:
let's make sure the typings path is correct (it probably needs some adjustments)